README.md

```diff
@@ -75,3 +85,46 @@ under `toxic`.
 This is similar to the bias madlibs technique, but has the advantage of using
 more realistic data. One can also use the model's performance on the original
 vs. fuzzed test set as a bias metric.
+
+## Dataset de-biasing
+
+TODO(jetpack): add dataset of additional examples as TSV with columns rev\_id,
+comment, and split. also add tool to recreate dataset from wikipedia dump.
+
+TODO(nthain,jetpack): upload new model trained on new dataset.
+
+This technique mitigates the dataset bias found in the **Dataset bias
+evaluation** section by determining the per-length-bucket deficit of non-toxic
+comments for each of the bias terms, and then sampling presumed non-toxic
+examples from Wikipedia articles.
+
+These new examples can then be added to the original dataset (to all splits,
+training as well as test). This allows (1) training a new model on the
+augmented, de-biased dataset and (2) evaluating the new and old models on the
+augmented test set.
```

Comment:
It's worth specifying that we will also evaluate new models on the old dataset. Something like 
"(2) and provide additional datasets on which to evaluate the performance of both the new and old models."

Reply:
doen